src/main/kotlin/pt/up/fe/ni/website/backend/model/Activity.kt

```diff
@@ -43,6 +43,8 @@ abstract class Activity(
     @field:NotBlank
     open var image: String,
 
+    val gallery: MutableList<String> = mutableListOf(),
```

Comment:
In order to use @ElementCollection the fetch type must be changed to Eager because by default it is Lazy, being Lazy the value must be obtained through a query or else it throws an exception, that's why my tests were failing en masse :p